draft-ietf-httpbis-semantics-latest.xml

```diff
@@ -606,22 +652,23 @@
    (<xref target="target.resource"/>). The request might also contain
    header fields (<xref target="header.fields"/>) for request modifiers,
    client information, and representation metadata,
-   content (<xref target="content"/>) to be processed
+   content (<xref target="content"/>) intended for processing
    in accordance with the method, and
-   trailer fields (<xref target="trailer.fields"/>) for metadata collected
-   while sending the content.
+   trailer fields (<xref target="trailer.fields"/>) to communicate information
```

Comment:
isn't this the same as: `<xref target="trailer.fields">trailer fields</xref>` ?

Reply:
AFAIU, it changes what part is hyperlinked. In any case, the latter style is the one we use throughout.